pymemcache/test/test_serde.py

```diff
@@ -76,3 +81,66 @@ class TestSerdePickleVersion2(TestCase):
 @pytest.mark.unit()
 class TestSerdePickleVersionHighest(TestCase):
     serde = PickleSerde(pickle_version=pickle.HIGHEST_PROTOCOL)
+
+
+@pytest.mark.parametrize("serde", [pickle_serde, CompressedSerde()])
+@pytest.mark.unit()
+def test_compressed_simple(serde):
```

Comment:
It would be nice to model these tests similar to the TestSerde class or even inherit from it.

Reply:
I was moving it away from having it inside a class to be more idiomatic pytest.   I migrated us to pytest way back in [2015](https://github.com/pinterest/pymemcache/pull/51) but never ripped out the class based tests.    I figured it made sense to make new tests more pytest like